pkg/cmd/stack/deploy.go

```diff
@@ -80,49 +83,34 @@ func Deploy(ctx context.Context, s *model.Stack, forceBuild, wait, noCache bool)
 	return err
 }
 
-func deploy(ctx context.Context, s *model.Stack, wait bool, c *kubernetes.Clientset) error {
+func deploy(ctx context.Context, s *model.Stack, wait bool, c *kubernetes.Clientset, config *rest.Config) error {
 	DisplayWarnings(s)
 	spinner := utils.NewSpinner(fmt.Sprintf("Deploying stack '%s'...", s.Name))
 	spinner.Start()
 	defer spinner.Stop()
 
-	addHiddenExposedPorts(ctx, s)
+	addHiddenExposedPortsToStack(ctx, s)
 
-	for name := range s.Services {
-		if len(s.Services[name].Ports) > 0 {
-			svcK8s := translateService(name, s)
-			if err := services.Deploy(ctx, svcK8s, c); err != nil {
-				return err
-			}
-		}
-	}
-
-	for name := range s.Volumes {
-		if err := deployVolume(ctx, name, s, c); err != nil {
-			return err
-		}
-		spinner.Stop()
-		log.Success("Created volume '%s'", name)
-		spinner.Start()
-	}
+	deployedSvcs := make([]string, 0)
+	deployedVolumes := make([]string, 0)
```

Comment:
why do you need to track `deployedVolumes`? We could deploy all volumes before entering the services loop, no?